ProjectPolice/police.py

```diff
@@ -23,22 +29,32 @@ def publish_to_notifier(message, channel):
         body=payload,
         properties=pika.BasicProperties(delivery_mode=2),
     )
+    return {
+        "success": True,
+        "data": {
+            "message": "message sent to Notifier",
+            "resources": payload,
+        },
+    }
 
 
 # Function to send message to Project Microservice
 def send_to_projectms(message):
     milestone_id = message["data"]["milestone_id"]
     project_id = message["data"]["project_id"]
     payload = format_message(milestone_id, project_id, message["data"])
-    url = format_url(project_id, milestone_id)
+    url = format_url(project_id, milestone_id, "penalise")
```

Comment:
url to update project milestone status should be `/projects/:id/milestone/:mid/status` so you dont need to format the url in this case, just change your existing path in your config.py file to
```python
PROJECT_STATUS_URL = (
    PROJECT_MS_URL + "/project/{project_id}/milestone/{milestone_id}/status"
)
```

then update the url in this `send_to_projectms` function like
```python
PROJECT_STATUS_URL.format(project_id=pid, milestone_id=mid)
```